src/render.ts

```diff
@@ -42,6 +42,11 @@ export type RenderOptions = {
 	*/
 	patchConsole?: boolean;
 
+	/**
+	Function to call after each render.
+	*/
+	onRender?: (renderTime: number) => void;
```

Comment:
I think it should be an object with renderTime property in case we want to add more things in the future.